changelog/argname.dd

```diff
@@ -0,0 +1,16 @@
+Stringify caller's function argument, analog to C's `#` stringification macro (except it's type safe).
```

Comment:
What exactly are you adding? `__traits(getCallerSource, a)` ? Or is it more than that?